editions/tw5.com/tiddlers/filters/syntax/Filter Run Prefix Legacy vs Named.tid

```diff
@@ -0,0 +1,41 @@
+created: 20230316151518640
+modified: 20230319150725216
+tags: [[Filter Run Prefix]]
+title: Filter Run Prefix Legacy vs Named
```

Comment:
It would be better to avoid using "vs" in the title. What is the title trying to convey?

I think the word "legacy" is misleading here. Better to describe them as "shortcut prefixes".

Also, as a matter of style "vs." should have a period.

Reply:
I'll rename it "shortcut prefixes". 

I did move this info to it's own tiddler, because I thought it may be needed to be transcluded several times. But I could avoid that for now. .. But the info should still be linkable as its own tiddler. So I'll want to keep it. 

May be "Filter Run Prefix Named and Shortcuts" may be better (but I don't like it very much. Just a gut feeling).
Initially I named it "Filter Run Prefix Old vs New" ... but that did not work either.

Reply:
will move the breadcrumbs macro into it's own PR.